src/manufacturer/manufacturerLandingPage.component.tsx

```diff
@@ -0,0 +1,243 @@
+import {
+  Box,
+  Button,
+  Grid,
+  Typography,
+  Link as MuiLink,
+  LinearProgress,
+} from '@mui/material';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import { Link, useLocation } from 'react-router-dom';
+import { useManufacturer } from '../api/manufacturer';
+
+import ManufacturerDialog from './manufacturerDialog.component';
+import React from 'react';
+
+function ManufacturerLandingPage() {
+  const location = useLocation();
+
+  const manufacturerId = location.pathname.replace(
+    '/inventory-management-system/manufacturer/',
+    ''
+  );
+
+  const { data: manufacturerData, isLoading: manufacturerDataLoading } =
+    useManufacturer(manufacturerId);
+
+  const [editManufacturerDialogOpen, setEditManufacturerDialogOpen] =
+    React.useState<boolean>(false);
+
+  return (
+    <Grid container>
+      <Grid sx={{ padding: '8px' }} item>
+        <Button
+          component={Link}
+          to={`/inventory-management-system/manufacturer/`}
+          sx={{ margin: '8px' }}
+          variant="outlined"
+        >
+          {manufacturerData ? 'Manufacturer table view' : 'Home'}
+        </Button>
+        <Button
+          disabled={!manufacturerData}
+          sx={{ margin: '8px' }}
+          variant="outlined"
+          onClick={() => {
+            setEditManufacturerDialogOpen(true);
+          }}
+        >
+          Edit
+        </Button>
+      </Grid>
+      {manufacturerData && (
+        <Grid item xs={12}>
+          <Grid container spacing={1} flexDirection="column">
+            <Grid item xs={12}>
+              <Typography sx={{ margin: 1, textAlign: 'center' }} variant="h4">
+                {manufacturerData.name}
+              </Typography>
+            </Grid>
+            <Grid container spacing={1} sx={{ px: '192px' }}>
+              <Grid
+                item
+                xs={12}
+                sx={{
+                  display: 'flex',
+                  flexDirection: 'row',
+                  alignItems: 'center',
+                  cursor: 'pointer',
```

Comment:
```suggestion
```
There are several of these below, they make it seem like something is clickable when it is not.